Userland/Libraries/LibWeb/WebIDL/DOMException.cpp

```diff
@@ -4,29 +4,27 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
-#include <LibWeb/HTML/Window.h>
+#include <LibWeb/HTML/Scripting/Intrinsics.h>
 #include <LibWeb/WebIDL/DOMException.h>
 
 namespace Web::WebIDL {
 
-JS::NonnullGCPtr<DOMException> DOMException::create(JS::Object& global_object, FlyString const& name, FlyString const& message)
+JS::NonnullGCPtr<DOMException> DOMException::create(JS::Realm& realm, FlyString const& name, FlyString const& message)
```

Comment:
Now that this is no longer in DOM/ you might want to move it out of the DOM commit, sorry about that 😅

Reply:
Eh. The refactor was a bit of a pain, but I don't really want to move these changes into their own commit, I'd rather keep them with the DOM.

Reply:
Fair enough, since you updated the commit message it's fine :+1: